admin/all-features-except

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+#
+# Usage: admin/all-features-except a,b,c crate
+#
+# Prints the set of all features declared by the given crate, minus
+# a, b and c.
+#
+# The output is decorated with `--no-default-features --features` meaning
+# it can be used directly with cargo, for example:
+#
+# $ cargo build $(admin/all-features-except std,logging rustls)
+
+set -e
+
+if [ $# -ne 2 ] ; then
+    echo "$0 <feature,feature,..> CRATE"
+    exit
+fi
+
+EXCEPT_FEATURES="$1"
+CRATE="$2"
+
+# gratuitously clone-and-hacked from
+# https://github.com/actix/actix-net/blob/master/justfile#L16-L22
```

Comment:
:pray: Thank you for sparing us having to wield `jq` like this from first principles :laughing:

Reply:
Personally I would feel more comfortable with a Python script over the use of shell script with `jq`, but I guess we have to make do?

Reply:
I would be open to that alternative. It would be easier to debug if it breaks 👍

Reply:
Replaced with python; much more readable!